apps/dashboard/src/components/variable/components/reorder-filter-item.tsx

```diff
@@ -17,7 +19,10 @@ type ReorderFilterItemProps = ComponentProps<typeof Reorder.Item<FilterWithParam
 
 export const ReorderFilterItem = (props: ReorderFilterItemProps) => {
   const { index, isLast, onRemove, onParamChange, value, ...rest } = props;
-  const filterDef = FILTERS.find((t) => t.value === value.value);
+  const isEnhancedDigestEnabled = useFeatureFlag(FeatureFlagsKeysEnum.IS_ENHANCED_DIGEST_ENABLED);
+  const liquidFilters = useMemo(() => getFilters(isEnhancedDigestEnabled), [isEnhancedDigestEnabled]);
```

Comment:
changed const `FILTERS` to a function `getFilters` that takes a feature flag; many places in the Dashboard required the change because of that